src/formatDate.js

```diff
@@ -50,7 +50,51 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
-}
+  const dateArr = date.split(fromFormat[fromFormat.length - 1]);
+  const separator = toFormat[toFormat.length - 1];
+  const obj = {};
+  let str = '';
+
+  for (let i = 0; i < fromFormat.length; i++) {
+    const element = fromFormat[i];
+
+    obj[element] = dateArr[i];
+  }
+
+  for (const item of toFormat) {
+    if (item === 'YY') {
+      if (fromFormat.includes(item)) {
+        str += obj.YY + separator;
+      } else {
```

Comment:
Try to avoid using nested if's and if/else statemenets. Check this [article ](https://medium.com/@timothydan/javascript-guard-clauses-64b999e3240) pls